mesh/v1alpha1/config.proto

```diff
@@ -1025,6 +1025,29 @@ message MeshConfig {
       // Optional. Controls the overall path length allowed in a reported span.
       // NOTE: currently only controls max length of the path tag.
       uint32 max_tag_length = 3;
+
+      // Optional. Specifies the configuration for exporting OTLP traces via HTTP.
+      // When empty, traces will be exported via gRPC.
+      OpenTelemetryTracingHttpService otlp_http = 4;
+    }
+
+    // Defines configuration for an OpenTelemetry tracing backend that accepts OTLP data via HTTP.
+    message OpenTelemetryTracingHttpService {
+      // REQUIRED. Specifies the path on the receiver for OTLP traces.
+      // The path is appended to the `service`
+      string path = 1;
```

Comment:
```suggestion
      string traces_path = 1;
```

I think I will call it `traces_path` because with just `path` it's not clear if it's related to traces/metrics or logs. From proto is clear because of the Message type, but when using the config it's not